src/audio/copier/copier.c

```diff
@@ -172,6 +200,8 @@ static int copier_free(struct processing_module *mod)
 		break;
 	}
 
+	if (cd)
+		rfree(cd->gtw_cfg);
```

Comment:
can we put line 203 and 204 to copier_ipcgtw_free? put here is not correct, since cd is true without gtw case.

Reply:
`cd->gtw_cfg ` is used in dai configuration so I'd rather move it to `copier_dai_free()`. Does that sound ok?

Reply:
that's fine if you confirmed there is no case for host.
and add if (cd->ipcgtw) in dai as well.